lib/mongoid/locking/selectable.rb

```diff
@@ -5,13 +5,15 @@ module Selectable # :nodoc:
       # its result.
       #
       # It is required when saving the same object multiple times in a row.
-      def atomic_selector
-        embedded? ? embedded_atomic_selector : root_atomic_selector
+      def atomic_selector(skip_lock_version: false)
+        embedded? ? embedded_atomic_selector : root_atomic_selector(skip_lock_version: skip_lock_version)
       end
 
       # https://github.com/mongodb/mongoid/blob/e03120a56894bc773dcf1e51209eb2f3e6f2b61f/lib/mongoid/selectable.rb#L55
-      def root_atomic_selector
-        { "_id" => id, "lock_version" => lock_version }
+      def root_atomic_selector(skip_lock_version: false)
```

Comment:
https://github.com/mongodb/mongoid/blob/7.2-stable/lib/mongoid/selectable.rb#L57

Reply:
also update comment to: https://github.com/mongodb/mongoid/blob/7.2-stable/lib/mongoid/selectable.rb#L57